gui/webdisplay/src/TWebWindow.cxx

```diff
@@ -749,10 +1031,9 @@ int ROOT::Experimental::TWebWindow::WaitFor(WebWindowWaitFunc_t check)
 
 int ROOT::Experimental::TWebWindow::WaitForTimed(WebWindowWaitFunc_t check, double timelimit)
 {
-   return fMgr->WaitFor(*this, check, true, timelimit);
+   return fMgr->WaitFor(*this, check, true, timelimit > 0 ? timelimit : GetOperationTmout());
```

Comment:
Please don't introduce meaning to magic value ("timeout < 0 means..."), esp without documenting it. If the called wants to have `GetOperationTmout()` they can pass it in.